build.gradle

```diff
@@ -108,11 +100,14 @@ subprojects {
                 auto_value_annotation: "com.google.auto.value:auto-value-annotations:${autoValueVersion}",
                 disruptor            : 'com.lmax:disruptor:3.4.2',
```

Comment:
Since everything has a variable for its version, should we add one here as well?

Reply:
I was focused on making sure that all dependencies are in the libraries to have a one place where we can update all deps. I think we can do that in the next PR.